Patches/GTest/Patch.cmake

```diff
@@ -0,0 +1,12 @@
+#+
+# This file is called as CMake -P script for the patch step of
+# External_GTest.cmake.
+# GTest_patch and GTest_source are defined on the command line along with the
+# call.
+#-
+
+message("Patching GTest")
```

Comment:
That may be, but there are several others that are also not `STATUS`... I vote for "leave it now, then fix all of them" or "fix all of them first and then I'll fix the PR also"...